core/src/main/java/greencity/controller/UserController.java

```diff
@@ -1270,4 +1271,23 @@ public ResponseEntity<Object> editAuthorities(@Valid @RequestBody UserEmployeeAu
         authorityService.updateEmployeesAuthorities(dto, principal.getName());
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Controller that update employees authorities.
+     *
+     * @param dto - UpdateEmployeeAuthoritiesDto.
+     * @author Nikita Korzh.
+     */
+    @ApiOperation(value = "Update an employee`s authorities")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
```

Comment:
Could we have a `404` response if the user is not found?

Reply:
done